client/app/utils/game-channel.js

```diff
@@ -0,0 +1,32 @@
+import { Socket } from 'vilive/utils/phoenix';
+import ENV from 'vilive/config/environment';
+
+let socket = null;
+let channel = null;
+
+export default {
+  init (token) {
+    socket = new Socket(ENV.yggdrasil.socket + '/socket',
+                        {params: {token: token}});
+    socket.connect();
+
+    // Now that you are connected, you can join channels with a topic:
+    channel = socket.channel("game:lobby", {"token": token});
+    channel.join()
+      .receive("ok", resp => { console.log("Joined successfully", resp); })
+      .receive("error", resp => { console.log("Unable to join", resp); });
```

Comment:
It's targeting the old code a the moment this is intentional, your changes have not been merged in. I branched off of json.api, so I would like to merge this back into json.api then rebase json api on master. Then I will update this with your current code and eliminate the web/static folder which still exists for now.

the end result for json.api will be two commits when I'm done one that adds json api support, and then one that adds ember with updated js code.